src/Raven.Server/Documents/Indexes/Persistence/Lucene/IndexReadOperation.cs

```diff
@@ -237,6 +237,8 @@ QueryResult CreateQueryResult(Document d)
                                     }
                                 }
 
+                                AddOrderByFields(query, document, ref d);
```

Comment:
DevelopmentHelper.ToDo review after Corax is merged

Reply:
done